tests/Routing/RoutingRouteTest.php

```diff
@@ -758,6 +758,25 @@ public function testResourceRouteNaming()
 		$this->assertTrue($router->getRoutes()->hasNamedRoute('bar'));
 	}
 
+	public function testControllerNaming()
```

Comment:
Please fix the cs. All repos must be separated by 2 blank lines in the tests, NOT 1.